autoload/airline/themes/term.vim

```diff
@@ -32,7 +32,7 @@ let s:R4 = [ '#E55345' , 9 ]
 let s:PA = [ '#94E42C' , 6 ]
 
 " Info modified
-let s:IM = [ '#40403C' , 238 ]
+let s:IM = [ '#40403C' , 7 ]
```

Comment:
why was this changed?

Reply:
i forgot to set that color when was submitting that colorscheme